features/step_definitions/alert_steps.rb

```diff
@@ -36,17 +36,17 @@ def create_entity_device(entity)
     find('label[for=device_errors]').click
 
     form.device_errors_value.set 2
-    form.sites.set_exact_multi "Mrs. Terry Goyette"
-    form.devices.set_exact_multi "Mr. Alphonso Witting"
-    form.roles.set_exact_multi "Institution Aric Smith Reader"
-    form.internal_users.set_exact_multi @user.email
+    form.sites.set_exact_multi "LAB SITE"
+    form.devices.set_exact_multi "INTERNAL DEVICE"
+    form.roles.set_exact_multi "LABORATORY"
+    # form.internal_users.set_exact_multi @user.email # FIXME: not listed!
```

Comment:
Just wondering: Instead of using fixed the names, we could store the models in the instance and refer to the values as `@site.name` etc.
That way we'd have faker provide different test values.

However, it seems the previous implementation also used fixed strings, but they were not specified at the creation... how did that work? Are these specs running with a fixed seed?

Reply:
Easy: they didn't work :shrug: